evadb/catalog/models/utils.py

```diff
@@ -36,8 +36,8 @@ def init_db(engine: Engine):
     if not database_exists(engine.url):
         logger.info("Database does not exist, creating database.")
         create_database(engine.url)
-        logger.info("Creating tables")
-        BaseModel.metadata.create_all(bind=engine)
+    logger.info("Creating tables")
```

Comment:
Just realized. We have function duplication here. We should be able to remove one instance.